node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+// import { read } from 'node:fs';
+import { readFile } from 'node:fs/promises';
+
+const [, , ...files] = process.argv;
+
+async function readAll(books) {
+  const proBooks = books.map(async (el) => {
+    return await readFile(el, 'utf8');
+  });
```

Comment:
This could be written more succinctly as:
`const proBooks = books.map((el) => readFile(el, 'utf8'));`
readFile returns a Promise, so you don't need to await it when mapping the array. You are awaiting it in the Promise.all.